src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Character Encoding</string>
```

Comment:
I was just using the text from the old code that I pulled out with git! :grinning: